OpenEdXMobile/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -1136,29 +1142,34 @@ private void requestAudioFocus() {
     private Runnable subtitleProcessesor = new Runnable() {
         @Override
         public void run() {
-            try{
+            try {
```

Comment:
Since i've touched/modified this code, so, I've also removed this unnecessary try-catch. Atleast thats what I do i.e. If I touch/modify some code, I remove the generic try-catch and refactor + test for any pitfalls.